roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -27,7 +27,7 @@ ExecStart=/usr/bin/rkt run \
         --volume etcd-ssl,kind=host,source={{ etcd_config_dir }},readOnly=true \
         --volume opt-cni,kind=host,source=/opt/cni,readOnly=true \
         --volume run,kind=host,source=/run,readOnly=false \
-        --volume usr-share-certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+        --volume usr-share-certs,kind=host,source={{ ssl_ca_dirs }},readOnly=true \
```

Comment:
we'll need to loop over ssl_ca_dirs -- ref: https://github.com/kubernetes-incubator/kargo/blob/master/roles/kubernetes/node/templates/kubelet-container.j2#L14